### ISSUE TITLE
Output Explaination

### ISSUE THREAD

**Issue by MayarOssama** [vdobrovolskii/wl-coref#36]
Hello, Thanks for sharing your work. Could you explain the output of the predict model?

TIA

**Comment by vdobrovolskii** [comment]
Hi!

Thanks for your interest.

Do you mean [predict.py](https://github.com/vdobrovolskii/wl-coref/blob/master/predict.py)?

The output is a file with one json per line. Each json contains the following keys:
```
cased_words:   a list of words in the document (each word is a string)
span_clusters: a list of coreferent clusters, each cluster is a list of spans, each span is a tuple of word indices (start, end).
    for instance, in "Jack and Jill went up the hill", [0, 1] will refer to "Jack" and [5, 7] will refer to "the hill".
word_clusters: a list of coreferent clusters, each cluster is a list of word indices.
    Basically it's the same as span_clusters, but with individual words instead. You would have 0 for "Jack" and 6 for "hill" here.

```
